pandas/core/series.py

```diff
@@ -505,7 +505,7 @@ def __init__(
                 data = data.copy()
         else:
             data = sanitize_array(data, index, dtype, copy)
-            data = SingleBlockManager.from_array(data, index, refs=refs)
+            data = SingleBlockManager.from_array(data, dtype, index, refs=refs)
```

Comment:
if `dtype` is your ListDtype, then data.dtype should be ListDtype at this point so the new argument should be unnecessary